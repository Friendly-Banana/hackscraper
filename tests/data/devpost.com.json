{"hackathons":[{"id":24295,"title":"UNIHACK 2025","displayed_location":{"icon":"globe","location":"Online"},"open_state":"upcoming","thumbnail_url":"//d112y698adiu2z.cloudfront.net/photos/production/challenge_thumbnails/003/300/709/datas/medium_square.png","analytics_identifier":"UNIHACK 2025 (24295)","url":"https://unihack2025.devpost.com/","time_left_to_submission":"Upcoming","submission_period_dates":"Mar 14 - 16, 2025","themes":[{"id":22,"name":"Open Ended"}],"prize_amount":"$\u003cspan data-currency-value\u003e8,600\u003c/span\u003e","prizes_counts":{"cash":3,"other":6},"registrations_count":2,"featured":false,"organization_name":"UNIHACK","winners_announced":false,"submission_gallery_url":"https://unihack2025.devpost.com/project-gallery","start_a_submission_url":"https://unihack2025.devpost.com/challenges/start_a_submission","invite_only":false,"eligibility_requirement_invite_only_description":null,"managed_by_devpost_badge":false}],"meta":{"total_count":1,"per_page":9,"fuzzy":true}}