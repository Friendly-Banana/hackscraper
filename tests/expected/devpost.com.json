[
    {
        "url": "https://unihack2025.devpost.com/",
        "image": "https://d112y698adiu2z.cloudfront.net/photos/production/challenge_thumbnails/003/300/709/datas/medium_square.png",
        "name": "UNIHACK 2025",
        "description": "Open Ended",
        "date": "Mar 14 - 16, 2025",
        "location": "Online"
    }
]